SUI_DZ/SUI_DZ/SoundViewModel.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  SoundViewModel.swift
+//  SUI_DZ
+//
+import AVFoundation
+
+/// Вью модель представления для управления воспроизведением аудио
+class PlayerViewModel: ObservableObject {
+    @Published public var maxDuration: TimeInterval = 0.0
+    @Published public var currentTime: TimeInterval = 0.0
+    @Published public var isPlaying = false
+    @Published public var progress: Float = 0.0
+    
+    private var player: AVAudioPlayer?
+    private var timer: Timer?
+    
+    /// Начать воспроизведение аудио
+    public func playSound(name: String) {
+        guard let audioPath = Bundle.main.path(forResource: name, ofType: "mp3") else { return }
+        do {
+            try player = AVAudioPlayer(contentsOf: URL(fileURLWithPath: audioPath))
```

Comment:
У банда можно спросить сразу URL, чтобы не делать лишнее.
```suggestion
        guard let audioUrl = Bundle.main.url(forResource: name, withExtension: "mp3") else { return }
        do {
            try player = AVAudioPlayer(contentsOf: audioUrl)
```